src/plugins/management/public/plugin.tsx

```diff
@@ -185,25 +185,11 @@ export class ManagementPlugin
         order: 3,
         async mount({ element, setBreadcrumbs }) {
           setBreadcrumbs([{ text: title }]);
-          const {
-            i18n,
-            docLinks,
-            notifications: { toasts },
-            settings,
-            theme,
-          } = core;
 
           ReactDOM.render(
             <KibanaRenderContextProvider {...core}>
-              <SettingsApplicationKibanaProvider
-                settings={{
-                  client: settings.client,
-                }}
-                theme={theme}
-                i18nStart={i18n}
-                {...{ i18n, toasts, docLinks }}
-              >
-                <SettingsApplication settingsStart={settings} />
+              <SettingsApplicationKibanaProvider {...core}>
```

Comment:
This is an example of type-widening, and why destructuring in the Provider is a decent practice.